### DISCUSSION TITLE
Possible that something is not right with the farming?

### DISCUSSION THREAD

**Discussion by henrikoo** [Chia-Network/chia-blockchain#14970] [Support]
Dear Community, 

We have set up a station in Switzerland where we are farming Chia Coins. Currently, we have almost 50,000 plots, which should yield around 60 Chias per month, based on our calculations. However, I have the feeling that something is not correct. Therefore, I would like to ask if it is possible that we are currently only earning 32 Chias per month with 50,000 plots, now few month. It should be 60 according to our calculations. Could this be due to settings? 

Please for support

**Comment by shaneo257** [comment]
per https://xchscan.com/chia-farming-calculator you could be looking at, at least 1 block per day, but this also not guaranteed as luck also comes into play, so it sounds about right

**Comment by heget** [comment]
Hi, I also experience strange behavior. My etw is 3 days and last month I got only 5 blocks insted of 10. And almost 20 days without hit the block. Do you have more farmers with your setup of 5 PB farm? ;)
